samples/web/content/apprtc/apprtc.py

```diff
@@ -333,7 +340,14 @@ def __str__(self):
 def get_memcache_key_for_room(host, room_id):
   return '%s/%s' % (host, room_id)
 
-def add_client_to_room(host, room_id, client_id, is_loopback):
+def does_room_exist(host, room_id):
+  key = get_memcache_key_for_room(host, room_id)
+  memcache_client = memcache.Client()
+  room = memcache_client.gets(key)
+  return room is not None
+
+def add_client_to_room(host, room_id, client_id,
+    is_loopback, intendedCallee = None):
```

Comment:
It's not clear if intendedCallee is a user_id, a gcm_id or an object. Maybe just callee_id? It might be worth storing it as a permissions list instead, since Room is set up for multiple clients right now it looks like. So e.g maybe do Room.clients and Room.connectedClients.